src/database/models/user.js

```diff
@@ -16,10 +16,12 @@ module.exports = (sequelize, DataTypes) => {
   User.init({
     firstName: DataTypes.STRING,
     lastName: DataTypes.STRING,
-    email: DataTypes.STRING
+    email: DataTypes.STRING,
+    userName: DataTypes.STRING,
+    password: DataTypes.STRING
   }, {
     sequelize,
     modelName: 'User',
   });
   return User;
```

Comment:
no validation implemented?
Email should be unique.
Character limits should be implemented

Reply:
password does not have to be required on the db validation since there is social sign up which won't have a password